synapse/http/proxyagent.py

```diff
@@ -198,8 +198,9 @@ def request(
                     self.http_proxy_creds.as_proxy_authorization_value(),
                 )
             # Cache *all* connections under the same key, since we are only
-            # connecting to a single destination, the proxy:
-            pool_key = ("http-proxy", self.http_proxy_endpoint)
+            # connecting to a single destination, the proxy
+            # The URL of proxy is not important for the key
```

Comment:
I think the original comment was clear enough.

```suggestion
            # connecting to a single destination, the proxy.
```